### PR TITLE
New Saved Jobs API

### PR DESCRIPTION
**Pull request by usbsnowcrash**
As the name implies

```ruby
saved_jobs = Cb.saved_jobs.get({ site: host_site, oauth_token: access_token })['data']
```

### CODE REVIEW

File: lib/cb/clients/saved_jobs.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        end
+
+        private
+
+        def uri_with_id(args)
+          "#{ Cb.configuration.uri_saved_job }/#{ args[:id] }"
+        end
+
+        def body(args)
+          body = Hash.new
```
Comment (Casao) [inline lib/cb/clients/saved_jobs.rb]: https://github.com/cbdr/ruby-style-guide#literal-array-hash

File: spec/cb/clients/saved_jobs_spec.rb [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+              with(:body => post_data.to_json, :headers => headers).
+              to_return(:status => 500, :body => error_response.to_json)
+          response = Cb::Clients::SavedJobs.create(job_id: 'J1234567891234567890', notes: 'these are my notes', oauth_token: 'token')
+          expect(response['errors'][0]).to eq(data)
+        end
+      end
+    end
+
+    context '#get' do
+      context 'asking for all cover letters' do
```
Comment (Casao) [inline spec/cb/clients/saved_jobs_spec.rb]: Saved jobs, not coverletter.

Comment (usbsnowcrash) [inline spec/cb/clients/saved_jobs_spec.rb]: copy pasta error...fixing

**Comment by usbsnowcrash** [comment]
I am ignoring the code climate errors for now.  The complex method in config is silly and the duplicate code warning might be addressed in the future as we refactor client down.

**Pull request merged**